### ISSUE TITLE
[Bug]: Doesn't work with Macaw's Paths and Pavings

### ISSUE THREAD

**Issue by CorneVgit** [Fuzss/blockrunner#4]
### Mod Loader (Required)

Fabric

### Minecraft Version (Required)

1.19.2

### Mod Version (Required)

v4.2.0

### Notes (Required)

Not sure if this classifies as a bug, but it felt like the most fitting option.

**Current behavior:**
I really like your mod, so I wanted to use it with Macaw's Paths and Pavings mod. Sadly there's no increased speed when moving over the paths or pavings from Macaw's mod after adding the path and paving blocks to the configuration file. The tooltip for the increased speed does show up on the paths and pavings blocks in REI so I'm pretty sure the issue isn't with the configuration.

I also tried this on the outdated 1.18.2 version and the same behavior occurs with the only difference being that in 1.18.2 the increased speed applies when jumping, probably because the character enters the air block above during the jump. In 1.19.2 the increased speed doesn't apply under any circumstances including jumping.

**Steps to reproduce:**
1. Add the path and pavings blocks from Macaw's mod to the config, for example the oak planks path block:
```json
{
  "minecraft:dirt_path": 1.35,
  "mcwpaths:oak_planks_path": 1.35
}
```
2. Place a few oak planks paths on the ground in a line.
3. Walk over them.

**Comment by MarioSMB** [comment]
This would be solved by #8, but in the meantime you can add a custom tag and populate it with Macaw's paths.

**Comment by Fuzss** [comment]
That issue is unrelated. The problem is Block Runner cannot handle those paths due to the blocks‘ small collision box.

**Comment by MarioSMB** [comment]
Apologies, it appeared to be a request for modded block support at first glance.

**Comment by Fuzss** [comment]
Should be fixed in v4.2.1.
